lib/beaker-pe/install/pe_utils.rb

```diff
@@ -448,6 +448,41 @@ def deploy_frictionless_to_master(host)
           end
         end
 
+        # Check for availability of required network resources
+        # @param [Array<Host>] hosts
+        # @param [Array<String>] network_resources, defaults to the value of options[:net_diag_hosts]
```

Comment:
param doc is out of date (no longer has a default).  Other than that I'm :+1:

Reply:
heh, good point.